packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -450,8 +454,8 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         }
 
         // Add data store's attributes to the summary.
-        const { pkg, isRootDataStore } = await this.getInitialSnapshotDetails();
-        const attributes = createAttributes(pkg, isRootDataStore, this.disableIsolatedChannels);
+        const { pkg } = await this.getInitialSnapshotDetails();
```

Comment:
Can you remove isRootDataStore from ISnapshotDetails since its not used anymore.